### DISCUSSION TITLE
Nested accordian

### DISCUSSION THREAD

**Discussion by sandeepsre** [mantinedev/mantine#2299] [Q&A]
How to make nested accordion. Typical structure screenshot attached
![screen](https://user-images.githubusercontent.com/16818395/187667032-c0a10d49-9d8e-4ce4-85e3-60b37d6c3b48.JPG)

**Comment by rtivital** [comment]
Render one Accordion inside another, do you have any issues with it?

**Reply by sandeepsre** [reply]
means render second accordion into  <Accordion.Panel>{Second Accordion}</Accordion.Panel>
   right?

**Reply by rtivital** [reply]
Correct

**Reply by sandeepsre** [reply]
thanks
